docs/developer-guide/joins/join-streams-and-tables.md

```diff
@@ -64,18 +64,18 @@ Joins and Windows
 -----------------
 
 ksqlDB enables grouping records that have the same key for stateful
-operations, like joins, into *windows*. You specify a retention period
-for the window, and this retention period controls how long ksqlDB waits
-for out-of-order records. If a record arrives after
-the window's retention period has passed, the record is discarded and
-isn't processed in that window.
+operations, like joins, into *windows*. At the moment, ksqlDB uses a fixed grace period of 24h:
```

Comment:
Without KLIP-36, it's not possible to specify the grace-period for stream-stream join windows (note, that there is not dedicated retention time, as KafkaStreams will set retentionTime = gracePeriod internally -- because the state stores are not exposed via IQ / pull queries, there is no need to allow for a larger retentionTime).